config/default.json

```diff
@@ -2,6 +2,7 @@
 	"scope": "springernature",
 	"packagesDirectory": "./packages",
 	"changelog": "HISTORY.md",
+	"CSSDirectoryStructure": ["10-settings", "20-functions", "30-mixins", "40-base", "50-components", "60-utilities"],
 	"allowExtends": false,
```

Comment:
this is good, but I would make one more small change
```
"CSSDirectoryStructure": {
	"scss": ["10-settings", "20-functions", "30-mixins", "40-base", "50-components", "60-utilities"]
}
```

This way we can remove the hardcoded `if (value === 'scss')` in `_build-new-package.js` and replace it with:

```
if (config.CSSDirectoryStructure && config.CSSDirectoryStructure[value])
```

then

```
config.CSSDirectoryStructure[value].forEach
```

Reply:
This puts everything in one place. We don't rely on the folder for sass being called scss, we make it easy for someone to be able to infer how this is constructed.